src/modal/classes.scss

```diff
@@ -5,8 +5,24 @@
 .iui-modal {
   @include iui-modal;
 
-  &-full-page {
-    @include iui-modal-full-page;
+  &-backdrop {
```

Comment:
Let's change the structure of `classes.scss` to follow @veekeys [suggestion shared here](https://github.com/iTwin/iTwinUI/pull/598#discussion_r857320738).  Personally I like how `classes.scss` looks currently, but Vykintas' suggestion will definitely make it easier to search and find classes.

Reply:
Changed